### PR TITLE
Common ExplorerContainer

### PR DESCRIPTION
**Pull request by AmndaPlant**
First step towards closing #645, creates a common ExplorerContainer component to hopefully reduce the amount of boilerplate needed to make a new explorer

### CODE REVIEW

File: client/src/IgocExplorer/IgocExplorer.js [hunk 1]
```diff
@@ -90,72 +82,34 @@ const scheme = {
...
-
-    const initialState = {
-      root: { ...initial_root_state, scheme_key },
-      [scheme_key]: {
-        grouping,
-        should_show_orgs_without_data: true,
-      },
-    };
+const IgocExplorer = ({ match }) => {
+  let grouping = _.get(match, "params.grouping");
```
Comment (ktw1016) [inline client/src/IgocExplorer/IgocExplorer.js:86]: Should short circuit this, like `const grouping = _.get(match, "params.grouping") || "portfolio"`. Although now that I see it, this is copy and pasted from old code. Still should avoid it

File: client/src/TagExplorer/TagExplorer.js [hunk 2]
```diff
@@ -109,40 +101,43 @@ function render_non_col_content({ node }) {
...
     this.debounced_set_query = _.debounce(this.debounced_set_query, 500);
   }
-  handleQueryChange(new_query) {
+
+  handleQueryChange = (new_query) => {
     this.setState({
-      _query: new_query,
-      loading: new_query.length > 3 ? true : undefined,
+      query: new_query,
+      loading: new_query.length.length > 3 ? true : undefined,
```
Comment (ktw1016) [inline client/src/TagExplorer/TagExplorer.js:116]: Is it a mistake .length twice here?

File: client/src/EstimatesComparison/EstimatesComparison.js [hunk 3]
```diff
@@ -101,7 +43,7 @@ const DetailedAmountsByDoc = ({ amounts_by_doc }) => {
         <div className="h6 heavy-weight">
           <TM k="doc_breakout_details" />
         </div>
-        <table className="table table-condensed">
+        <table className="table tale-condensed">
```
Comment (ktw1016) [inline client/src/EstimatesComparison/EstimatesComparison.js:46]: Was this a mistake?

**Comment by AlexCLeduc** [comment]
Found a bug on estimates comparison, click any of these checkboxes and they disappear, and make the "by org vs. by type" toggle lose its selection

Screen Shot 2020-08-28 at 11 36 59 AM

**Comment by AmndaPlant** [comment]
Yeah I'm honestly not sure what could be causing this bug, it looks like somehow the hierarchy is going from "org" to "undefined" but no clue why

**Comment by ktw1016** [comment]
Closing in favor of #763

**Pull request closed without merging**